contracts/CO30_Felix-Emerson_SameDayDelivery.zargabad/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,14 @@
+<font color='#E3D310'>PRIMARY</font>
+Deliver specific supplies requested by IDAP to the designated IDAP camps within the open tents.
+
+<font color='#595305'>SECONDARY</font>
+Eliminate any hostile forces found in the area.
+
+<font color='#E3D310'>RESTRICTIONS</font>
+- Contractors are NOT allowed to fire weapons or engage in combat within IDAP facilities.
```

Comment:
I would say something more like "Contractors are NOT allowed to brandish weapons within IDAP facilities

If the players have their weapons away, but then get engaged while inside an IDAP facility, they can return fire. They just can not have weapons in hand or initiate combat within the facility.